### PR TITLE
Allow retrieving the list of articles from the extension

### PR DESCRIPTION
**Pull request by tarikeshaq**
Basically moves #43 to the `content.js` and sends requests to backend to get the articles!

Tested it and it works!

Gonna leave this open until @toast682 and @Bebo0 get some time to give it a look.

Feel free to modify this!

Note that currently this is based on top of #47 .  Github is smart enough to rebase this to `main` once that one gets merged 😄 

fetch

### CODE REVIEW

File: frontend/content.js [hunk 1]
```diff
@@ -4,56 +4,20 @@ chrome.runtime.onMessage.addListener(function (message, sender, sendResponse) {
...
-            },
-            {
-                ID: 6,
-                TITLE: 'We Tried It: What I Learned About My Rescue Puppy Through a Dog DNA Screening',
-                URL: 'https://people.com/pets/we-tried-it-dog-dna-test-for-rescue-puppy/'
-                // Votes []Vote ignore votes for now
-            }];
-        sendResponse({listArticles: listArticles});
+    } else if (message.request === 'getList') {
+        getData('http://localhost:8090/articles').then(data => sendResponse({ listArticles: data }));
```
Comment (lhao03) [inline frontend/content.js:8]: should `http://localhost:8090` be something like an env file? so its not hardcoded.

Comment (tarikeshaq) [reply frontend/content.js:8]: Yeah - I think should go into the `manifest.json`, I believe we can define values there for local development vs production. Which is exactly what we need 👀

**Comment by potatokim** [comment]
Looks good. Learned a few things. Thanks Tarik!

**Review by potatokim — APPROVED** [review]
Looks good. Learned a few things. Thanks Tarik!

**Review by lhao03 — APPROVED** [review]

**Review by Bebo0 — APPROVED** [review]
Tested it locally, the extension accesses the articles added to the db properly. Gj!

**Pull request merged**